app/src/main/java/com/google/maps/android/compose/StreetViewActivity.kt

```diff
@@ -40,21 +40,31 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.dp
 import com.google.android.gms.maps.StreetViewPanoramaOptions
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.Status
 import com.google.maps.android.compose.streetview.StreetView
 import com.google.maps.android.compose.streetview.rememberStreetViewCameraPositionState
 import com.google.maps.android.ktx.MapsExperimentalFeature
 import kotlinx.coroutines.launch
+import com.google.maps.android.StreetViewUtils.Companion.fetchStreetViewData
```

Comment:
double-checking: is this class being pulled in via the `maps-compose-utils` dependency?